public/src/bc5SupportDev/java/com/revenuecat/purchases/models/PurchaseOption.kt

```diff
@@ -0,0 +1,32 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.android.billingclient.api.ProductDetails
+import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.ComparableData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.RawValue
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+data class PurchaseOption(
+    val pricingPhases: List<PricingPhase>,
+    val tags: List<String> = listOf(),
+    val token: String // "offerToken" used for purchasing
+) : Parcelable {
+    val isBasePlan: Boolean
+        get() = pricingPhases.size == 1
+
+    val isFreeTrial: Boolean
+        get() = pricingPhases[0].priceAmountMicros == 0L
+
+    // idk if we can do something smarter here or if this is even valuable
+     val isIntroPrice: Boolean
+        get() = !isFreeTrial
+
```

Comment:
What are we going to use those helpers for? I think they'll be useful for rendering the paywal, right>
But then it would be nice to get the FreeTrial phase to say soemthing like "Try one week for free@, or "Pay only 5.99 the first two months and then..."
So maybe they could return a `PricingPhase?` instead? (For intro price it might have to be a list, if we want to support the 5 phases that could exist)

Reply:
i like the new naming ⭐ 

let's table this discussion. i think adding nice helper functions is something we can think about as we finalize the API and its use cases. i added a TODO to remember to come back to it, or maybe we can keep a section in the document somewhere?